metadata.rb

```diff
@@ -12,6 +12,8 @@
 depends          'chef-vault', '~> 1.3'
 depends          'ulimit', '~> 0.3'
 depends          'xml', '~> 1.2'
+# build-essential >=8.0.0 requires chef 12.5+
+depends          'build-essential', '< 8.0.0'
```

Comment:
This still needed since we are using 12.17.44 in the Vagrantfile and since we should probably update the Gemfile to match?